### DISCUSSION TITLE
Show issue/PR creator on card face in Project Board view

### DISCUSSION THREAD

**Discussion by gregsdennis** [community/community#69051] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

In the project board view, the cards show the assigned member, but I'd like the ability to show the creator.  For now I have to manually assign the creator to all of the PRs.

<img width="262" alt="image" src="https://github.com/community/community/assets/2676804/2de1836d-a6bf-4ede-97cd-76209b8248a6">

This would help identify who is seeking approval.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by joaorosera** [comment]
Response #1

**Reply by gregsdennis** [reply]
Yeah?  I looked at that, and it seems that I'm following that guidance.  Did I miss something?

**Comment by Abkeyev99** [comment]
Click "Marked as answer" Thanks so good :)
